scss/style.scss

```diff
@@ -0,0 +1,24 @@
+//sass modules
+@use 'sass:color';
+
+//reset styles
+@import 'base/_reset';
```

Comment:
You can omit the `_` when importing scss partials

Reply:
Didn't know that, thanks for the tip!